orangecontrib/text/widgets/owconcordance.py

```diff
@@ -223,11 +227,15 @@ def set_width(self):
                 QItemSelectionModel.SelectCurrent | QItemSelectionModel.Rows)
 
     def set_corpus(self, data=None):
+        self.closeContext()
+
```

Comment:
While at it, would it not make sense to remove the empty line? Or is this intentional?